test/functional/interface_zmq_dash.py

```diff
@@ -131,35 +130,39 @@ def run_test(self):
             self.activate_dip8()
             self.nodes[0].sporkupdate("SPORK_17_QUORUM_DKG_ENABLED", 0)
             self.wait_for_sporks_same()
-            # Create an LLMQ for testing
-            self.quorum_type = 100  # llmq_test
-            self.quorum_hash = self.mine_quorum()
+            self.activate_v19(expected_activation_height=900)
+            self.log.info("Activated v19 at height:" + str(self.nodes[0].getblockcount()))
+            self.move_to_next_cycle()
+            self.log.info("Cycle H height:" + str(self.nodes[0].getblockcount()))
+            self.move_to_next_cycle()
+            self.log.info("Cycle H+C height:" + str(self.nodes[0].getblockcount()))
+            self.move_to_next_cycle()
+            self.log.info("Cycle H+2C height:" + str(self.nodes[0].getblockcount()))
+
+            self.mine_cycle_quorum(llmq_type_name='llmq_test_dip0024', llmq_type=103)
+
             self.sync_blocks()
             self.wait_for_chainlocked_block_all_nodes(self.nodes[0].getbestblockhash())
             # Wait a moment to avoid subscribing to recovered sig in the test before the one from the chainlock
             # has been sent which leads to test failure.
             time.sleep(1)
             # Test all dash related ZMQ publisher
-            self.test_recovered_signature_publishers()
+            #self.test_recovered_signature_publishers()
             self.test_chainlock_publishers()
             self.test_governance_publishers()
             self.test_getzmqnotifications()
-            self.test_instantsend_publishers(False)
-            self.activate_dip0024()
-            self.log.info("Activated DIP0024 at height:" + str(self.nodes[0].getblockcount()))
-            # Test for CL 8 blocks after dip24 activation because along with dip24, the BLS scheme is activted
-            self.generate_blocks(8)
+            self.test_instantsend_publishers()
             self.wait_for_chainlocked_block_all_nodes(self.nodes[0].getbestblockhash())
-            self.test_instantsend_publishers(False)
+            self.test_instantsend_publishers()
             # At this point, we need to move forward 3 cycles (3 x 24 blocks) so the first 3 quarters can be created (without DKG sessions)
             self.move_to_next_cycle()
-            self.test_instantsend_publishers(False)
+            self.test_instantsend_publishers()
             self.move_to_next_cycle()
-            self.test_instantsend_publishers(False)
+            self.test_instantsend_publishers()
             self.move_to_next_cycle()
-            self.test_instantsend_publishers(False)
+            self.test_instantsend_publishers()
             self.mine_cycle_quorum()
-            self.test_instantsend_publishers(True)
+            self.test_instantsend_publishers()
```

Comment:
do we still need all of these cycles?

Reply:
We don't want to alter initial test logic, just to force deterministic islocks.